frontend/src/router/useRecentVisit.ts

```diff
@@ -14,12 +14,22 @@ const MAX_HISTORY = 3;
 
 export function useRecentVisit() {
   const route = useRoute();
-  const authStore = useAuthStore();
-  const recentVisit = useLocalStorage(
-    `${STORAGE_KEY}.${authStore.currentUserId ?? 0}`,
+  const currentUser = useCurrentUserV1();
+  let recentVisit = useLocalStorage(
```

Comment:
how about to try `useDynamicLocalStorage`?

Reply:
Really cool!